<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>321</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>Форма изменения</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>231</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="priceLabel">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="price"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>231</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>231</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Цвет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="color"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>231</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="imageUrlLabel">
        <property name="text">
         <string>URL картинки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="imageUrl"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>231</width>
       <height>96</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="description"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>231</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="brandLabel">
        <property name="text">
         <string>Бренд</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="brand"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>231</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="weightLabel">
        <property name="text">
         <string>Вес</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weight"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QScrollArea" name="responseOutputArea">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>69</y>
      <width>311</width>
      <height>451</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>449</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="outputTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="getByIdButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>179</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Get by id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getAllButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>79</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Get all</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="deleteByIdSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>481</y>
      <width>44</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>280</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="updateByIdSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>44</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="getByIdSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>179</y>
      <width>44</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteByIdButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>481</y>
      <width>89</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete by id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateByIdButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>380</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Update by id</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
